happi/cli.py

```diff
@@ -0,0 +1,110 @@
+"""
+Module for a command line interface to happi
+"""
+# cli.py
+# NOTE: Does not work from any environment, must have pcds_conda activated
+## Can this script activate/deactivate pcds_conda upon use?
```

Comment:
If you use the `setup.py` that I linked, the command will be made available once you enter `pcds_conda`

Reply:
I think I set this up, just made a commit (was working locally) and I need to try creating a new conda environment on the ps machines to see if this works

Reply:
Also I added the options for `--version` and `--verbose`. I made this commit so I could move to the ps machines and try to create a development conda package and test the `--version` option. The way I'm using `--verbose` may be slightly off, specifically at line 13:
`logger = logging.getLogger(__name__)`
I know this depends on how I run `cli.main()`